test/e2e/framework/framework.go

```diff
@@ -279,3 +279,26 @@ func UpdateDeployment(kubeClientSet kubernetes.Interface, namespace string, name
 
 	return nil
 }
+
+// UpdateConfigmap updates the ingress-nginx config
```

Comment:
Could we make this `updateNginxConfigMap`? So we have a sense of whose/which ConfigMap is updated?

Reply:
Nice, but if we pass in the name of the configMap, it'd still be _somewhat_ specific since the method needs to know that we're talking about the namespace `ingress-nginx`. 

I think I'll go with `updateNginxConfigMap`, and if we find out that we need a generic one for the temporary deployments maybe - we can update :)